### PR TITLE
Ports cargo borgs to Effigy

### PR DESCRIPTION
**Pull request by dekunutkid**
## About The Pull Request
Pretty self explanatory, this ports cargo borgs from Skyrat with code by GoldenAlpharex and art by Serijas#9275, I claim no ownership of any part of this.
Proof of function:
image
image
image
## Why It's Good For The Game
More borg varieties are cool, more ways to play the game through the very limited but powerful capabilities of borgs.
## Changelog
:cl:
add: Added cargo borgs, in all your favourite flavours!
/:cl:

### CODE REVIEW

File: local/code/modules/cargo/packs/cargoborg/code/robot_items.dm [hunk 1]
```diff
@@ -0,0 +1,482 @@
+/// CARGO BORGS ///
+#define CYBORG_FONT "Consolas"
+#define MAX_PAPER_INTEGRATED_CLIPBOARD 10
```
Comment (distributivgesetz) [inline local/code/modules/cargo/packs/cargoborg/code/robot_items.dm:3]: undefine these at the bottom of the file

**Review by distributivgesetz — CHANGES_REQUESTED** [review]

File: local/code/modules/cargo/packs/cargoborg/code/robot_items.dm (hunk 1 above)
Comment (dekunutkid) [reply local/code/modules/cargo/packs/cargoborg/code/robot_items.dm:3]: I'm sorry, what does this mean? I'm trying to find other examples of undefining as reference but I can't seem to be able to.

Comment (distributivgesetz) [reply local/code/modules/cargo/packs/cargoborg/code/robot_items.dm:3]:
```
#undef CYBORG_FONT
...
```

File: local/code/modules/cargo/packs/cargoborg/code/robot_items.dm [hunk 2]
```diff
@@ -0,0 +1,482 @@
+/// CARGO BORGS ///
+#define CYBORG_FONT "Consolas"
```
Comment (lessthnthree) [inline local/code/modules/cargo/packs/cargoborg/code/robot_items.dm:2]:
```suggestion
#define CYBORG_FONT "Ubuntu Mono"
```

**Review by lessthnthree — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/silicon/robot/robot_model.dm [hunk 3]
```diff
@@ -157,7 +157,6 @@
 			var/obj/item/gun/energy/gun = module
 			if(!gun.chambered)
 				gun.recharge_newshot() //try to reload a new shot.
-
```
Comment (distributivgesetz) [inline code/modules/mob/living/silicon/robot/robot_model.dm:160]: Don't edit upstream files unless you have to, please

**Review by distributivgesetz — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/silicon/robot/robot_model.dm (hunk 3 above)
Comment (dekunutkid) [reply code/modules/mob/living/silicon/robot/robot_model.dm:160]: I would love to remove this file from the PR, but i can't figure out how.

Comment (distributivgesetz) [reply code/modules/mob/living/silicon/robot/robot_model.dm:160]: Add a newline

**Review by distributivgesetz — APPROVED** [review]

**Pull request merged**